src/main/resources/db/migration/common/V1_214__update_security_roles_data.sql

```diff
@@ -0,0 +1,22 @@
+UPDATE security_role
+set role_name='REQUESTER'
+WHERE rol_id = 2;
+
+UPDATE security_role
+set role_name='APPROVER'
+WHERE rol_id = 1;
+
+UPDATE security_role
+set role_name='JUDGE'
+WHERE rol_id = 3;
+
+UPDATE security_role
+set role_name='TRANSCRIBER'
+WHERE rol_id = 4;
+
+UPDATE security_role
+set role_name='LANGUAGE SHOP USER'
```

Comment:
Should these last 2 roles have screaming snake case names like you have set the permissions?